doc/faq.rst

```diff
@@ -6,267 +6,518 @@
+    Please note that Auto-sklearn will stop when either the time limit or the number of
+    models termination condition is reached.
+
+.. collapse:: <b>Why does the final ensemble contains only a dummy model?</b>
+
+    This is a symptom of the problem that all runs started by Auto-sklearn failed. Usually, the issue
+    is that the runtime or memory limit were too tight. Please check the output of
+    ``sprint_statistics()`` to see the distribution of why runs failed. If there are mostly crashed
+    runs, please check the log file for further details. If there are mostly runs that exceed the
+    memory or time limit, please increase the respective limit and rerun the optimization.
+
+.. collapse:: <b>Auto-sklearn does not use the specified amount of resources?</b>
+
+    Auto-sklearn wraps scikit-learn and therefore inherits its parallelism implementation. In short,
+    scikit-learn uses two modes of parallelizing computations:
+
+    1. By using joblib to distribute independent function calls on multiple cores.
+    2. By using lower level libraries such as OpenMP and numpy to distribute more fine-grained
+       computation.
+
+    This means that Auto-sklearn can use more resources than expected by the user. For technical
+    reasons we can only control the 1st way of parallel execution, but not the 2nd. Thus, the user
+    needs to make sure that the lower level parallelization libraries only use as many cores as
+    allocated (on a laptop or workstation running a single copy of Auto-sklearn it can be fine to not
+    adjust this, but when using a compute cluster it is necessary to align the parallelism setting
+    with the number of requested CPUs). This can be done by setting the following environment
+    variables: ``MKL_NUM_THREADS``, ``OPENBLAS_NUM_THREADS``, ``BLIS_NUM_THREADS`` and
+    ``OMP_NUM_THREADS``.
+
+    More details can be found in the `scikit-learn docs <https://scikit-learn.org/stable/computing/parallelism.html?highlight=joblib#parallelism>`_.
+
+Other
+=====
 
-Which meta-features are used for meta-learning?
------------------------------------------------
+.. collapse:: <b>Model persistence</b>
 
-We do not have a user guide on meta-features but they are all pretty simple and can be found
-`in the source code <https://github.com/automl/auto-sklearn/blob/master/autosklearn/metalearning/metafeatures/metafeatures.py>`_.
+    *auto-sklearn* is mostly a wrapper around scikit-learn. Therefore, it is
+    possible to follow the
+    `persistence Example <https://scikit-learn.org/stable/modules/model_persistence.html>`_
+    from scikit-learn.
 
-How is the meta-data generated?
--------------------------------
+.. collapse:: <b>Vanilla auto-sklearn</b>
 
-Auto-sklearn 1.0
-~~~~~~~~~~~~~~~~
+    In order to obtain *vanilla auto-sklearn* as used in `Efficient and Robust Automated Machine Learning
+    `<https://papers.nips.cc/paper/5872-efficient-and-robust-automated-machine-learning>`_
```

Comment:
This fails with the doc tests, I assume it just a case of removing the backtick before the `<https ...>` and perhaps making sure it's not spread out over a linebreak.